### ISSUE TITLE
BuildLog and ViceLog txt file mistake?

### ISSUE THREAD

**Issue by arekbr** [Swoffa/SublimeKickAssemblerC64#45]
Hello again

I can't understand what I do wrong

This is compile command
compileCommand = javaCommand+" cml.kickass.KickAssembler \"${build_file_base_name}.${file_extension}\" -log \"${kickass_output_path}/${build_file_base_name}_BuildLog.txt\" -o \"${kickass_output_path}/${kickass_compiled_filename}\" -vicesymbols -showmem -symbolfiledir \"${kickass_output_path}\" ${kickass_args}"

arguments
-log \"${kickass_output_path}/${build_file_base_name}_BuildLog.txt\" generate file_BuildLog.txt

debug command is:

debugCommand = "\"${kickass_debug_path}\" -logfile \"${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" -moncommands \"${kickass_output_path}/${build_file_base_name}_MonCommands.mon\" ${kickass_debug_args} \"${kickass_output_path}/${kickass_compiled_filename}\""

need to load 
-logfile \"${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" - file_ViceLog.txt

???? I don't have this file after compilation ends. I can't override this args because debug command still want to load file_ViceLog.txt and myown -logfile\"${kickass_output_path}/${build_file_base_name}_BuildLog.txt\" do nothing. 

I hope I explain clearly my problem.

How to fix it or what I do wrong? How I can remove unwanted args?

in short version - I need this command as run_debug cmd. 
c64debugger.exe -autojmp -layout 11 -breakpoints bin/examplefilename.dbg -symbols bin/examplefilename.vs -wait 2500 -pass -prg bin/examplefilename.prg

Can You help me? 

Thank You
kind regards
Arek

**Comment by Swoffa** [comment]
Hi, thanks for feedback.
I have on my list to add proper support for C64Debugger, and also support for KickAssembler 5.
Unfortunately i am extra short on time right now. I will look at Pr #46 from @BroneCk, or try some other quick fix for you whenever i get a bit time.

**Comment by Swoffa** [comment]
PR #47 merged now. 
It helps in 2 ways;
- Uses separate default command for c64debugger
- Has settings for detailed control of run and debug commands
  - `kickass_run_command_x64`
  - `kickass_debug_command_x64`
  - `kickass_debug_command_c64debugger`
  - `kickass_run_command_c64debugger`
